oscarlab/graphene
# [Build] Ignore .cache.mk file in gsgx

## Description of the changes 

Companion to https://github.com/oscarlab/graphene-sgx-driver/pull/29

Building the gsgx module on newer kernels creates a temporary .cache.mk file. This is cleaned up by 'make clean', but not ignored by git, causing CI failures on the gitignore check. Add this to .gitignore.



## How to test this PR? 

Run the gitignore check on gsgx module, in CI or manually.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1884)
